packages/graphback-datasync/src/DataSyncPlugin.ts

```diff
@@ -85,6 +87,10 @@ export class DataSyncPlugin extends GraphbackPlugin {
       }
     })
 
+    if (dataSyncModelCount === 0) {
+      console.error("No DataSync Models detected, ensure that your models are properly annotated.")
```

Comment:
Maybe use `this.logWarning` to keep consistent with the other plugins ([including this one](https://github.com/aerogear/graphback/blob/071c2744aed20eea9414ac9c8e445c620e1545d9/packages/graphback-datasync/src/DataSyncPlugin.ts#L31))